src/frontend/src/flows/dappsExplorer/index.ts

```diff
@@ -58,7 +61,11 @@ const dappTemplate = ({
       rel="noopener noreferrer"
     >
       <div class="c-action-list__icon" aria-hidden="true">
-        <img src=${logo} alt=${name} loading="lazy" />
+        <img
+          src=${logo.replace("/img/showcase/", "/icons/")}
+          alt=${name}
+          loading="lazy"
```

Comment:
arent' these icons not lazy anymore?

Reply:
I thought the same, but actually not. This is the explorer (i.e. a new page) whereas the icons you're referring to are the ones in the "teaser", i.e. the marquee/button on the management page.